README.md

```diff
@@ -43,6 +43,55 @@ Optional parameters:
   - `az,type`
   - `id,publicIP,launchTime`
 
+### /ec2/all ###
+Search all ec2 instances.
+
+### /rds/all ###
+Search all RDS resources.
+
+### /dynamo ###
+
+Search tables in DynamoDB.
+
+Optional parameters:
+
+- `q` / `query`: OGNL expression used to filter. Example:
+  - `id > 100 && readCapacityUnits > 10`
+- `s` / `sort`: OGNL expression used as a ``Comparable`` to sort (default to no ordering). Example:  
+  - `s=itemCount`
+- `l` / `limit`: maximum number of records to return. Example:
+  - `l=10`
+- `f` / `field`: comma-separated list of fields to display(defaults to all). Examples:
+  - `f=tableName,tableStatus`
+  - `f=tableName`
+
+Sample response (the ``readCapacityUnits`` and ``writeCapacityUnits`` are pulled out of ``provisionedThroughput`` which allows users to sort with these attributes):
+
+```
+{
+  "tableName": "user_table",
+  "attributeDefinitions": "[{AttributeName: foo,AttributeType: S}]",
+  "tableStatus": "ACTIVE",
+  "keySchema": "[{AttributeName: foo,KeyType: HASH}]",
+  "creationDateTime": 14887890,
+  "numberOfDecreasesToday": 0,
+  "readCapacityUnits": 1,
+  "writeCapacityUnits": 1,
+  "tableSizeBytes": 0,
+  "itemCount": 0,
+  "tableArn": "arn:aws:dynamodb:us-east-1:xxxxx:table/user_table,
+  "provisionedThroughput": "{NumberOfDecreasesToday: 0,ReadCapacityUnits: 1,WriteCapacityUnits: 1}"
+}
+```
+
+Sample query: show readCapacityUnits and tableName of all tables sorted by readCapacity:
+``/dynamo?s=readCapacityUnits&&f=readCapacityUnits,tableName``
+
+
+
+### /dynamo/all  ###
+Search all tables in DynamoDB.
+
```

Comment:
Unrelated, but could you add the related RDS docs? It should be pretty straight forward since I believe we only allow for `rds/all`